Resource.js

```diff
@@ -129,12 +135,20 @@ module.exports = function(app, route, modelName, model) {
               })
             });
             break;
+          case 204:
+            // Convert 204 into 200, to preserve the empty result set.
+            if (req.method !== 'DELETE') {
```

Comment:
Agreed. Just change the responses to a 204 and do an empty object or empty array depending on the required response.